spec/models/reports/repository_spec.rb

```diff
@@ -454,7 +469,7 @@ def refresh_views
 
         it "creates cache keys" do
           repo = Reports::Repository.new(facility_1, periods: Period.month("June 1 2019")..Period.month("Jan 1 2020"))
-          cache_keys = repo.send(:cache_entries, :controlled).map(&:cache_key)
+          cache_keys = repo.schema.send(:cache_entries, :controlled).map(&:cache_key)
```

Comment:
These changes to testing the schema were the one thing that threw me here. It seems like this approach to testing would have worked before but maybe wasn't a critical distinction until now.

Reply:
yeah these cache tests will probably go away soon, so I'm not too worried about them.